src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController+DynamicTableViewModel.swift

```diff
@@ -458,7 +458,7 @@ extension ExposureDetectionViewController {
 		let activeTracing = state.risk?.details.activeTracing ?? .init(interval: 0)
 		return DynamicTableViewModel([
 			riskDataSection(cells: [
-				.riskContacts(text: AppStrings.ExposureDetection.numberOfContacts, image: UIImage(named: "Icons_RisikoBegegnung")),
```

Comment:
This was a cleanup together with removing the `ExposureDetection_NumberOfContacts` from the stringsdict?

Reply:
Yep